Automattic/jetpack
# Protect: Add WAF header stat cards

#### Description
Adds `StatCard` components to the `FirewallHeader`

#### Changes proposed in this Pull Request:

* Makes the `StatCard` component a default `js-packages` component export
* Applies the `StatCard` component to the `FirewallHeader`
* Retrieves and display WAF blocked requests when applicable

#### Other information:

#### Jetpack product discussion


- 1202232924268038-as-1203282049665710

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:






* Checkout this branch
* Start your Jurassic Tube without a scan level upgrade on a site with a history of blocked requests
* Activate Protect and proceed to the Firewall tab
* Ensure that the `StatCard` components are disabled, display the placeholder value, and a "Paid feature" label
* Upgrade your plan
* Return the Firewall page and verify that the `StatCard` components are now active and display an appropriate value for each period
* Switch to mobile view and ensure that the structure and styling line up with the provided designs
* Verify that there is no regression in app functionality introduced

#### Screenshots

Disabled:
<img width="1156" alt="Screen Shot 2022-12-20 at 14 28 28" src="https://user-images.githubusercontent.com/43220201/208779089-3ab50384-eb06-4d1c-b79a-feb850d98941.png">

Active:
<img width="1165" alt="Screen Shot 2022-12-20 at 14 22 19" src="https://user-images.githubusercontent.com/43220201/208779057-034f0aaa-9f55-4419-8792-70ecf8045df9.png">

Mobile:
<img width="205" alt="Screen Shot 2022-12-20 at 14 28 07" src="https://user-images.githubusercontent.com/43220201/208779144-2dd89d29-9693-4cd8-b706-5ff7c03000e4.png">

<img width="204" alt="Screen Shot 2022-12-20 at 14 22 37" src="https://user-images.githubusercontent.com/43220201/208779134-56e6a57a-501a-4a0b-b6a2-147064d999d0.png">
